src/client/routes/vessels-router.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { Route, withRouter, Link } from "react-router-dom";
+
+import VesselDetails from "./../components/VesselDetails";
+
+const VesselRouter = props => {
+  return (
+    <React.Fragment>
```

Comment:
Nice to see you know about `Fragments`, but there is absolutely no point in using them here, since you already have a `<div>` wrapping everything.

Reply:
Removed fragment